pkg/repositories/database_integration_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
why add this?

Reply:
This is automatically generated by `goimports` (goimports -w pkg/*) 
Could we add this? So we don't need to manually delete it when running `goimports`

Reply:
thank you